_layouts/default.liquid

```diff
@@ -13,6 +13,7 @@
       <meta http-equiv="refresh" content="3; url={{ redirect }}">
     {% endif %}
     {% include head.liquid %}
+    <script src="{{ '/assets/js/shortcut-key.js' | relative_url }}"></script>
```

Comment:
Does it affect if this script is added to the end of the body with the other scripts?

Reply:
Nope ! Good idea. Moved it to the bottom of the body.